src/components/SelectionList/components/Footer.tsx

```diff
@@ -32,12 +33,12 @@ function Footer<TItem extends ListItem>({footerContent, confirmButtonConfig, add
                 <Button
                     success
                     large
-                    style={[styles.w100, confirmButtonConfig?.style]}
-                    text={confirmButtonConfig?.text}
-                    onPress={confirmButtonConfig?.onConfirm}
+                    style={[styles.w100, style]}
+                    text={text}
+                    onPress={onConfirm}
                     pressOnEnter
                     enterKeyEventListenerPriority={1}
-                    isDisabled={confirmButtonConfig?.isDisabled}
+                    isDisabled={isDisabled}
```

Comment:
Nitpick: I'd say that this is less informative now. I'd rename `style` to `confirmButtonStyle` (`text` to `confirmButtonText` etc)
But it can just be me so you can ignore